test/update.test.js

```diff
@@ -144,7 +144,7 @@ describe('updateAll', function() {
       Product.update({id: id}, newData, function(err, result) {
         testUtil.hasError(err, result).should.not.be.ok();
         testUtil.hasResult(err, result).should.be.ok();
-        var newRev = result[0].rev;
```

Comment:
If `.rev` was previously correct but now it is not, doesn't that make this a breaking change?